packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -796,11 +881,29 @@ class Scaffold extends StatefulWidget {
     this.resizeToAvoidBottomPadding,
     this.resizeToAvoidBottomInset,
     this.primary = true,
+    this.extendBody = false,
```

Comment:
might be good to make this true by default...

Reply:
I set this value to “false” because I was afraid to break existing projects. Backward compatibility 💯

Reply:
I agree. If we made extendBody true by default it would break some existing layouts.